### PR TITLE
[RLlib][RLModule] Fixed the learning test regression introduced via learner_api in pendulum

### PR DESCRIPTION
**Pull request by kouroshHakha**
This PR also tries to match the metrics keys to the old stack, so that side by side benchmarking is possible. 

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/algorithms/ppo/ppo_base_learner.py [hunk 1]
```diff
@@ -14,6 +15,9 @@ def __init__(self, *args, **kwargs):
         if self.hps.entropy_coeff_schedule:
             raise ValueError("entropy_coeff_schedule is not supported in Learner yet")
 
+        # TODO (Kourosh): This needs to be native tensor variable to be traced.
```
Comment (avnishn) [inline rllib/algorithms/ppo/ppo_base_learner.py:18]: does it need to be traced? only if we need grads fed through it, which I dont think is the case

Comment (avnishn) [reply rllib/algorithms/ppo/ppo_base_learner.py:18]: oops i take it back

**Review by avnishn — APPROVED** [review]
lgtm pending a tensorboard or something like that

**Comment by kouroshHakha** [comment]
> lgtm pending a tensorboard or something like that

The better curve is from after this PR :) 

tf2, before vs. after change
Screen Shot 2023-03-27 at 5 33 49 PM

**Review by zhe-thoughts — APPROVED** [review]
Approved for merging into master

**Pull request merged**